packages/components/src/Button/Button.tsx

```diff
@@ -162,8 +162,9 @@ function ButtonInternals({ label, icon, size = "base" }: ButtonProps) {
       {icon && <Icon name={icon} size={size} />}
       <Typography
         element="span"
-        textCase="uppercase"
+        textCase="capitalize"
```

Comment:
I know I wrote the ticket but thinking about we shouldn't actually use `capitalize` here. I'd like to follow the same guidelines we have for [mobile Button](https://atlantis-mobile.jobber.dev/?path=/docs/atlantis-actions-button--button) around "proper" title-casing – it does add more work on the implementation-in-product side as we'll have a bunch of labels to tweak, but we should be able to catch those instances fairly easily.

Reply:
Good call out - I removed the `textCase` variable and updated the `.mdx` file to match the guidelines we have for mobile.  Like we discussed, once we are updating the layouts in the future ticket, we will look at `<Button>` components in JO, Dev Center, etc. I don't mind doing that when the time comes 👍